DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -579,7 +577,7 @@ def export_getResourceInfo( self , resource = None ):
     """ Send the storage element resource information
 """
 
-    conn , error, userDict = self.__irodsClient( )
+    conn , error, _ = self.__irodsClient()
```

Comment:
I don't know if it is a general good practice or not, but I like to name the unused variables "_varName" so "_userDict" in that case. Pylint knows that it will not be used, and it still lets the reader know what is actually unpacked